recipes/crowcpp-crow/all/conanfile.py

```diff
@@ -42,12 +42,12 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("boost/1.81.0")
+        self.requires("boost/1.83.0")
         if self.version == "0.2":
-            self.requires("openssl/1.1.1s")
+            self.requires("openssl/[>=1.1 <4]")
         if Version(self.version) >= "1.0":
             if self.options.with_ssl:
-                self.requires("openssl/1.1.1s")
+                self.requires("openssl/1.1.1v")
```

Comment:
Why does 0.2 allow for version range but no >=1.0?

Reply:
@RubenRBS 
I met following execution error with OpenSSL/3.x.x:
```
terminate called after throwing an instance of 'boost::wrapexcept<boost::system::system_error>'
  what():  context: unknown option (digital envelope routines) [asio.ssl:50331817]
```

It seems to be caused by OpenSSL/3.x.x providers.
https://wiki.openssl.org/index.php/OpenSSL_3.0#Providers

Reply:
@toge you still can use ranges, but in that case: `openssl/[>=1.1 <3]`

Reply:
@uilianries 
Thanks a lot!

Reply:
Weird that an old version supports openssl 3, but newer ones dont!